src/mesh/NodeDB.cpp

```diff
@@ -349,6 +349,10 @@ void NodeDB::init()
     if (channelFileCRC != crc32Buffer(&channelFile, sizeof(channelFile)))
         saveWhat |= SEGMENT_CHANNELS;
 
+    if (!devicestate.node_remote_hardware_pins) {
+        memset(devicestate.node_remote_hardware_pins, 0, sizeof(devicestate.node_remote_hardware_pins));
+    }
```

Comment:
you can also directly assign meshtastic_ModuleConfig_RemoteHardwareConfig_init_zero

Reply:
or [...]_init_default for that matter

Reply:
Can't reinitialize arrays in C :-(